vowpalwabbit/comp_io.h

```diff
@@ -21,19 +21,19 @@ class comp_io_buf : public io_buf
  public:
   std::vector<gzFile> gz_files;
 
-  virtual int open_file(const char* name, bool stdin_off, int flag = READ);
```

Comment:
are we sure these shouldn't be virtual?

Reply:
Yes, override is what we need to mark these correctly